src/ray/gcs/accessor.h

```diff
@@ -183,6 +183,12 @@ class JobInfoAccessor {
   /// \param is_pubsub_server_restarted Whether pubsub server is restarted.
   virtual void AsyncResubscribe(bool is_pubsub_server_restarted) = 0;
 
+  /// Increment and get next job id. This is not idempotent.
+  ///
+  /// \param done Callback that will be called when request successfully.
+  /// \return Status
+  virtual Status AsyncGetNextJobID(const ItemCallback<int> &callback) = 0;
```

Comment:
I think we should use `JobID` instead of `int` on interfaces.

Reply:
The return value will be passed into Java interface in int format, so we can keep it as int, to avoid transform it over and back.

Reply:
I don't agree with you. The performance cost of int -> JobID -> bytes -> JobId (java) should be subtle. Let's keep the code as readable as possible.

Reply:
That makes sense.